src/BaGet/Startup.cs

```diff
@@ -46,6 +47,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 }
             }
 
+            app.UsePathBase(Configuration.Get<BaGetOptions>().PathBase);

```

Comment:
Does this throw if `PathBase` is `null`? If it does, please add a null check.

Reply:
This option does not throw on `null`.